tensorflow_addons/optimizers/tests/weight_decay_optimizers_test.py

```diff
@@ -212,129 +173,159 @@ def sgdw_update_numpy(param, grad_t, slot_vars, learning_rate, momentum, weight_
     return param_t, slot_vars
 
 
-@test_utils.run_all_in_graph_and_eager_modes
-class AdamWTest(OptimizerTestBase):
+optimizer_AdamW = weight_decay_optimizers.AdamW
```

Comment:
We don't need an alias there, it actually hurts the readability. Please use `weight_decay_optimizers.AdamW` directly.